package.json

```diff
@@ -7,8 +7,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.12",
-    "@gen3/guppy": "^0.9.0",
     "@material-ui/core": "^4.11.0",
+    "@pcdc/guppy": "github:chicagopcdc/guppy#pcdc_dev",
```

Comment:
@bobaekang since we are using the master branch for the release we should probably have master here instead of pcdc_dev and have pcdc_dev on your local dev env. Saying that, do you need me to merge some changes of guppy from pcdc_dev into master to have this be compatible I guess?

Reply:
Great point. I just think that it's okay for now as long as we 1) merge the changes to guppy master branch and 2) update the portal dependencies before the code freeze. I have some ideas to improve guppy components source code as well, which might be suitable for the next guppy version.

Reply:
@grugna In fact, when we merge pcdc_dev to master next time, we should add to it an appropriate git tag and use it to versioning portal dependencies.

Reply:
sounds good, do you have any preferences on the tag name?

Reply:
@grugna It should simply match our release version. E.g. when we merge `pcdc_dev` to `master` later this month, we should tag it as `0.1.3` or `v0.1.3`, whichever seems better.